packages/react-app/src/views/BuilderProfileView.jsx

```diff
@@ -48,9 +59,39 @@ export default function BuilderProfileView({ serverUrl, mainnetProvider, address
 
   useEffect(() => {
     fetchBuilder();
+    const asyncUpdateEvents = async () => {
+      const res = await getWithdrawEvents(builderAddress);
+      setWithdrawEvents(res);
+    };
+    asyncUpdateEvents();
     // eslint-disable-next-line
   }, [builderAddress]);
 
+  useEffect(() => {
+    if (!builder) {
+      return;
+    }
+    const stream = builder.stream;
+
+    const cap = ethers.utils.parseUnits(stream.cap);
```

Comment:
This breaks the page (WSOD) if the builder doesn't have an stream

We should check it first

Reply:
fixed for users without streams, and users with streams but indexer hasn't processed the stream yet.

For the indexer scenario I'm just checking that `stream.cap` is falsy, either `0` or `undefined`. I think that's going to work with the current indexer implementation, right?